doc/developer_guide/coding-style.rst

```diff
@@ -133,9 +154,9 @@ Some important details:
 
   #) There are up to four major sections in each interface description: function
      description, parameter description and type, return value and type, and
-     exceptions. The function description is required, all other sections only
-     need be provided if they are applicable for the code being documented.
-     The formatting for each section is slightly different:
+     raises (exceptions). The function description is required, all other
+     sections only need be provided if they are applicable for the code being
+     documented. The formatting for each section is slightly different:
 
          .. tabularcolumns:: |l|L|
```

Comment:
Not your fault, but should there be two lower-case "l" for "left" in tabular environment?

Reply:
No - I've just checked (https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html) and the L indicates that Sphinx should automatically wrap the content.